daemon/src/taker.rs

```diff
@@ -31,7 +31,7 @@ const CONNECTION_RETRY_INTERVAL: Duration = Duration::from_secs(5);
 #[derive(Parser)]
 struct Opts {
     /// The IP address of the other party (i.e. the maker).
-    #[clap(long, default_value = "127.0.0.1:9999")]
```

Comment:
Imho if you remove this you will need to adapt `start-all.sh`. Also, CI smoke test might fail without a default value.

Reply:
Yep just saw CI failing. Will fix!